app/bundles/FormBundle/Entity/Form.php

```diff
@@ -735,6 +746,22 @@ public function setInKioskMode($inKioskMode)
         $this->inKioskMode = $inKioskMode;
     }
 
+    /**
+     * @return mixed
+     */
+    public function getInContactTab()
```

Comment:
Does it really return `mixed`? The `$inContactTab` property is annotated as `bool`. (the same in the setter)

If it will be bool, it would be nice to call this method `isInContactTab()` as it makes it more human friendly :)